code/modules/clothing/spacesuits/void.dm

```diff
@@ -1,14 +1,131 @@
 
-//NASA Voidsuit
-/obj/item/clothing/head/helmet/space/nasavoid
+//Voidsuits
+/obj/item/clothing/head/helmet/space/voidsuit
 	name = "NASA Void Helmet"
-	desc = "A high tech, NASA Centcom branch designed, dark red space suit helmet. Used for AI satellite maintenance."
-	icon_state = "void"
+	desc = "A retro space suit helmet designed by a branch at NASA Centcom. This one is in red."

```

Comment:
You missed a `m` in Centcomm. Also is NASA even still a thing in ss13 lore?

(This also applies to all the other descriptions below)

Reply:
That is how it originally was spelled in the description, but I can change it.

Reply:
I would imagine NASA to be dead by this point by lore. I would normally think it still operational this far in the future but I don't see mega corporations taking over our world just yet sooo...

Reply:
I agree, but the people who want them to be NASA suits are way more vocal then the people who question it, so, *shrugs*